Gruntfile.js

```diff
@@ -18,22 +20,29 @@ module.exports = (grunt) => {
   const TARGET_MIN = 'vexflow-min.js';
 
   // Used for eslint and docco
-  const SOURCES = ['src/*.ts', 'src/*.js', '!src/header.js'];
+  const SOURCES = ['./src/*.ts', './src/*.js', '!./src/header.js'];
 
-  // Take all test files in 'tests/' and build TARGET_TESTS
-  const TARGET_TESTS = path.join(BUILD_DIR, 'vexflow-tests.js');
-  const TEST_SOURCES = ['tests/vexflow_test_helpers.js', 'tests/mocks.js', 'tests/*_tests.js', 'tests/run.js'];
+  // Take all test files in 'tests/' and build TARGET_TESTS_BROWSER
+  const TARGET_TESTS_BROWSER = 'vexflow-tests.js';
+  const TARGET_TESTS_NODE = 'vexflow-tests-node.js';
+  const TEST_SOURCES = [
+    './tests/vexflow_test_helpers.js',
```

Comment:
@0xfe you commented in #872

> I'd certainly like to see what tests/vexflow_test_helpers.js would look like.

do mean migrating to ts? in the one file approach it should be possible without duplicating the file.

@wassertim what do you think?